Formula/bazarr.rb

```diff
@@ -0,0 +1,70 @@
+class Bazarr < Formula
+  include Language::Python::Virtualenv
+
+  desc "Companion to Sonarr and Radarr for managing and downloading subtitles"
+  homepage "https://www.bazarr.media"
+  url "https://github.com/morpheus65535/bazarr/releases/download/v1.0.3/bazarr.zip"
+  sha256 "540a5c4d80110ba35a566c17829cd6f31444db7ade7c6b9d0603eccaaed8a628"
+  license "GPL-3.0-only"
+  head "https://github.com/morpheus65535/bazarr.git", branch: "master"
+
+  depends_on "gcc"
+  depends_on "numpy"
+  depends_on "python@3.9"
+
+  uses_from_macos "zlib"
+
+  resource "webrtcvad-wheels" do
```

Comment:
Is there a reason you didn't include `lxml`? https://github.com/morpheus65535/bazarr/blob/v1.0.3/requirements.txt#L2

If you do add it, you probably will need `uses_from_macos` for `libxml2` and `libxslt`

Reply:
Just an oversight, nothing more. I'll try that out.